src/tools/nnfusion/templates/dxcompute/Direct3DWinNN/Direct3DWinNN.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
Seems we do not need to include .sln, vcxproj.user, vcxproj.filters files

Reply:
> Seems we do not need to include .sln, vcxproj.user, vcxproj.filters files

then we could remove these files from templates